apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/SyncOnboardingCompanion.tsx

```diff
@@ -484,12 +484,15 @@ const SyncOnboardingCompanion: React.FC<SyncOnboardingCompanionProps> = ({
   useEffect(() => {
     if (seedPathStatus === "recover_seed" && recoverRestoreStaxPath) {
       const [pathname, search] = recoverRestoreStaxPath.split("?");
+      console.log("#SyncOnboardingCompanion:: pathname", pathname);
+      console.log("#SyncOnboardingCompanion:: search", search);
 
-      history.push({
-        pathname,
-        search: search ? `?${search}` : undefined,
-        state: { fromOnboarding: true },
-      });
+      setStepKey(StepKey.Applications);
+      // history.push({
+      //   pathname,
+      //   search: search ? `?${search}` : undefined,
+      //   state: { fromOnboarding: true },
+      // });
```

Comment:
if you comment this code you will just never redirect to the Recover restore flow, no ?

Reply:
it is in draft. please dont review.